internal/bminventory/inventory.go

```diff
@@ -1847,20 +1847,19 @@ func (b *bareMetalInventory) updateNonDhcpNetworkParams(updates map[string]inter
 	var err error
 	*machineCidr, err = network.CalculateMachineNetworkCIDR(apiVip, ingressVip, cluster.Hosts)
```

Comment:
you can't ignore all the errors, for example https://github.com/openshift/assisted-service/blob/master/internal/network/machine_network_cidr.go#L45-L49